world_universities_and_domains.json

```diff
@@ -132066,5 +131810,246 @@
       "student.hult.edu"
     ],
     "country": "United States"
+  },
+  {
+    "web_pages": [
+      "https://www.bs-erlangen.de/"
+    ],
+    "name": "Erlangen State Vocational School",
+    "alpha_two_code": "DE",
+    "state-province": null,
+    "domains": [
+      "bs-erlangen.de"
+    ],
+    "country": "Germany"
+  },
+  {
+    "web_pages": [
+      "https://www.huas.edu.cn/"
+    ],
+    "name": "Hunan University of Arts and Science",
+    "alpha_two_code": "CN",
+    "state-province": null,
+    "domains": [
+      "huas.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://www.tyust.edu.cn/"
+    ],
+    "name": "TaiYuan University of Science and technology",
+    "alpha_two_code": "CN",
+    "state-province": null,
+    "domains": [
+      "tyust.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://www.lyun.edu.cn/"
+    ],
+    "name": "Longyan University",
+    "alpha_two_code": "CN",
+    "state-province": null,
+    "domains": [
+      "lyun.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "http://www.wsyu.edu.cn/"
+    ],
+    "name": "Wuchang Shoyi University",
+    "alpha_two_code": "CN",
+    "state-province": null,
+    "domains": [
+      "wsyu.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://psnacet.edu.in/"
+    ],
+    "name": "PSNA College of Engineering and Technology",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "psnacet.edu.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.iiitbh.ac.in/"
+    ],
+    "name": "Indian Institute of Information Technology, Bhagalpur",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "iiitbh.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "https://www.iiitsurat.ac.in/"
+    ],
+    "name": "Indian Institute of Information Technology, Surat",
+    "alpha_two_code": "IN",
+    "state-province": null,
+    "domains": [
+      "iiitsurat.ac.in"
+    ],
+    "country": "India"
+  },
+  {
+    "web_pages": [
+      "http://bvucoepune.edu.in/"
```

Comment:
Invalid website, the correct one is `https://www.bvuniversity.edu.in/`

Reply:
Forget about this one. I cannot properly fact-check this :D. Need to verify with Emilia